gulpfile.js

```diff
@@ -81,10 +82,15 @@ function buildTask(name, stageList) {
   });
 };
 
+function buildExperimentTask(name, experimentFile) {
+  experimentFiles.push(experimentFile);
+  buildTask(name, [{name: 'input', options: {data: experimentFile}}, 'fileToBuffer', 'bufferToString', 'doExperiment']);
+};
+
 /*
  * Some example pipelines.
  */
-buildTask('html', [{name: 'input', options: {data: options.file}}, 'fileToBuffer', 'bufferToString', 'jsonParse', 'HTMLWriter', {name: 'writeStringFile', options: {filename: 'result.html.html'}}]);
+buildExperimentTask('html', 'tasks/html.exp');
```

Comment:
* Moved tasks/html.exp to tasks.erlnmyr.
 * Using erlnmyr extension.
 * Removed buildExperimentTask, leaving erlnmyr-register.js to a separate patch.